src/server/browserType.ts

```diff
@@ -41,11 +40,11 @@ export type LaunchOptions = BrowserArgOptions & {
 };
 
 export interface BrowserType {
-  executablePath(): string;
+  executablePath(): (string|null);
```

Comment:
Let's always return a string and throw if it was not set?

Reply:
This is technically the only way to check if there's a browser to be launched; the usecase is valid, and I don't like supporting it with an exception.

WDYT?